client/lib/gsuite/new-users.ts

```diff
@@ -176,9 +177,11 @@ const validateNewUserMailboxIsUnique = (
  * Adds a duplicate error to each mailBox with a duplicate mailbox
  */
 const validateNewUsersAreUnique = ( users: GSuiteNewUser[] ): GSuiteNewUser[] => {
-	const mailboxesByCount: { [ mailbox: string ]: number } = countBy(
-		users.map( ( { mailBox: { value: mailBox } } ) => mailBox )
-	);
+	const mailboxesByCount = users.reduce( ( workingCount: Record< string, number >, user ) => {
+		const mailboxName = user.mailBox.value;
+		workingCount[ mailboxName ] = 1 + ( workingCount[ mailboxName ] ?? 0 );
+		return workingCount;
```

Comment:
I would add blank lines as otherwise this block of code is a bit difficult to read:
```suggestion

		workingCount[ mailboxName ] = 1 + ( workingCount[ mailboxName ] ?? 0 );

		return workingCount;
```

Reply:
Done! 16db851